docker/0.5.0/fedimintd-mainnet-tls/docker-compose.yaml

```diff
@@ -0,0 +1,74 @@
+# See the .env file for config options
+
+version: "5.0"
+
+services:
+  traefik:
+    image: "traefik:v2.10"
+    container_name: "traefik"
+    command:
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.websecure.address=:443"
+      - "--certificatesresolvers.myresolver.acme.tlschallenge=true"
+      - "--certificatesresolvers.myresolver.acme.storage=/letsencrypt/acme.json"
+    ports:
+      - "443:443"
+    volumes:
+      - "letsencrypt_data:/letsencrypt"
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+
+  fedimintd:
+    image: fedimint/fedimintd:v0.5.0
+    volumes:
+      - fedimintd_data:/data
+    ports:
+      - "0.0.0.0:8173:8173"
+    environment:
+      - FM_DEFAULT_BITCOIN_RPC_KIND=${FM_BITCOIN_RPC_KIND}
+      - FM_DEFAULT_BITCOIN_RPC_URL=${FM_BITCOIN_RPC_URL}
+      - FM_BITCOIN_NETWORK=bitcoin
+      - FM_BIND_P2P=0.0.0.0:8173
+      - FM_P2P_URL=fedimint://${FM_DOMAIN}:8173
+      - FM_BIND_API=0.0.0.0:8174
+      - FM_API_URL=wss://${FM_DOMAIN}/ws/
+      - FM_REL_NOTES_ACK=0_4_xyz
+      - FM_EXTRA_DKG_META=meta_override_url=https://${FM_DOMAIN}/meta.json
+    restart: always
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.fedimintd.loadbalancer.server.port=8174"
+      - "traefik.http.routers.fedimintd.rule=Host(`${FM_DOMAIN}`) && Path(`/ws/`)"
+      - "traefik.http.routers.fedimintd.entrypoints=websecure"
+      - "traefik.http.routers.fedimintd.tls.certresolver=myresolver"
+
+  guardian-ui:
+    image: fedimintui/guardian-ui:0.4.2
```

Comment:
Should we use 0.4.4 from the ui repo?

Reply:
There isn't one afaict.

Reply:
Ahh shoot, I don't see a tag in dockerhub even though we cut an v0.4.4 in the UI repo.

Created an issue in the UI repo to automate publishing similar to fedimint/fedimint. https://github.com/fedimint/ui/issues/593